src/IntercomCompanies.php

```diff
@@ -33,6 +33,37 @@ public function update($options)
         return $this->create($options);
     }
 
+    /**
+     * Attaches a Contact to a Company.
+     *
+     * @see    https://developers.intercom.io/reference#attach-contact-to-company
+     * @param  string $contactId
+     * @param  array $options
+     * @return stdClass
+     * @throws Exception
+     */
+    public function attachContact($contactId, $options)
```

Comment:
I have mixed feelings about something, so hope to get your view on this too 😄 On one hand, this API wrapper is super-lightweight and we generally just take one parameter for the id and other for the payload (options). In this case, this call requires the options to contain a specific parameter, so I think being more specific here would be helpful, ie

```suggestion
    public function attachContact(string $contactId, string $companyId)
    {
      $options = [ "id" => $companyId ];
      ...
    }
```

What do you think about that change? I'm happy with keeping it as you did for consistency, but also happy to move to what I proposed 😄

Reply:
You raise a good point, and I agree that the function should take a specific parameter for the `$companyId`. In order to provide as much flexibility as possible, I was thinking of keeping `$options` as a parameter and using `array_merge` like so:
```suggestion
    public function attachContact(string $contactId, string $companyId, $options)
    {
        $options = array_merge($options, ["id" => $companyId]);
        ...
    }
```
What are your thoughts on this?

Reply:
Makes total sense! That would definitely make it more flexible in case we add additional parameters in the future 😄 Let's do that, but adding a default for `$options`:

```suggestion
    public function attachContact(string $contactId, string $companyId, array $options = [])
    {
        $options = array_merge($options, ["id" => $companyId]);
        ...
    }
```